lib/cisco_node_utils/radius_server.rb

```diff
@@ -39,6 +39,24 @@ def initialize(name, instantiate=true)
       end
       @name = name
 
+      if platform == :ios_xr
```

Comment:
Right now the Nexus initialize ignores the provided auth_p and acct_p values. That doesn't seem quite right to me.